src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -1248,6 +1248,14 @@ def filter_and_paginate(
     # Initialization and configuration
     # --------------------------------
 
+    def _run_migrations(self) -> None:
+        if self.skip_migrations or handle_bool_env_var(
```

Comment:
This is a breaking change for anyone who relied on that env variable only having to be set previously, but we can probably ignore that. At least our official helm chart sets the value to `True`.